public_html/index.php

```diff
@@ -436,7 +428,7 @@
       <div class="col-sm-6">
         <h2 id="get-started" class="text-white">Get started in minutes</h2>
         <p class="text-white-50">Nextflow lets you run sanger-tol pipelines on virtually any computing environment.</p>
-        <p class="text-white-50">All sanger-tol genomics pipelines come with built-in support for
+        <p class="text-white-50">All sanger-tol genomic pipelines are modifiable with the
```

Comment:
We're using nf-core for pipelines in general, not just the genomics ones (even though everything we do is genomics)
```suggestion
        <p class="text-white-50">All sanger-tol pipelines are modifiable with the
```